WordPress/gutenberg
# Introduce prependHTTPS url util

## What?


Adds `prependHTTPS` util function to url package.

## Why?


See https://github.com/WordPress/gutenberg/pull/47312#issue-1550557064.

Also based on feedback from @tyxla in https://github.com/WordPress/gutenberg/pull/47312#issuecomment-1406237583.

## How?


Utilises existing `prependHTTP` function implementation, replacing `http` with `https` where appropriate.

Duplicates all tests.

## Testing Instructions





See test cases and run

```
npm run test:unit packages/url/src/test/index.js
```

### Testing Instructions for Keyboard


## Screenshots or screencast
